modules/@apostrophecms/piece-type/ui/apos/components/AposImageCropper.vue

```diff
@@ -233,24 +257,66 @@ export default {
       this.isUpdatingCropperCoordinates = false;
     },
     /**
-     * Return wether the cropper coordinates have changed or not.
+     * Sets cropper coordinates via its API.
+     */
+    setCropperCoordinates (coordinates) {
+      this.$refs.cropper.setCoordinates(coordinates);
+    },
+    /**
+     * Gets focal point relative position inside the stencil
+     * and emits it to update `x` and `y` as percentages,
+     * or as `null` if outside it.
+     */
+    updateFocalPointCoordinates () {
+      const { focalPoint } = this.$refs;
+
+      const focalPointSize = this.getFocalPointSize();
+
+      const x = (
+        focalPoint.offsetLeft +
+        focalPointSize.halfWidth -
+        this.stencilCoordinates.left
+      ) / this.stencilCoordinates.width;
+      const y = (
+        focalPoint.offsetTop +
+        focalPointSize.halfHeight -
+        this.stencilCoordinates.top
+      ) / this.stencilCoordinates.height;
+
+      const sanitizeCoordinates = ({ x, y }) =>
```

Comment:
Use `function` and move to the end of the method for more readable top-down structure.